.tekton/openshift-builds-bundle-test.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/redhat-openshift-builds/release?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "1"
+    pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request" &&
+      target_branch == "main" &&
+      (
+        files.all.exists(x, x.matches('pipleines/konflux-build.yaml')) ||
+        files.all.exists(x, x.matches('.tekton/openshift-builds-bundle-test.yaml'))
+      )
+  labels:
+    appstudio.openshift.io/application: openshift-builds-release
+    appstudio.openshift.io/component: openshift-builds-bundle-test
+    pipelines.appstudio.openshift.io/type: build
+  name: openshift-builds-bundle-test
+  namespace: rh-openshift-builds-tenant
+spec:
+  params:
+  - name: git-url
+    value: https://github.com/redhat-openshift-builds/operator.git
+  - name: revision
+    value: main
+  - name: output-image
+    value: quay.io/redhat-user-workloads/rh-openshift-builds-tenant/openshift-builds-release:{{revision}}
```

Comment:
Should this denote operator image??

Reply:
Technically this won't create a issue because we are just trying to test the build and push to registry and the image is of no use. 
But logically these doesn't feel right and now that I am thinking in future we can use this image for further test scenarios. 
I will create new repositories for each component.